suite-native/navigation/src/components/ScreenContent.tsx

```diff
@@ -7,11 +7,14 @@ import { Box } from '@suite-native/atoms';
 
 type ScreenContentProps = {
     children: ReactNode;
+    footer: ReactNode;
     isScrollable: boolean;
     customVerticalPadding: number;
     customHorizontalPadding: number;
 };
 
+const KEYBOARD_OFFSET = 225;
```

Comment:
What is this specific number?

Reply:
removed in https://github.com/trezor/trezor-suite/commit/a2d3117fe98d90bb542c026df2b20fa1beb7c583